src/cow-react/modules/limitOrders/containers/TradeButtons/limitOrdersTradeButtonsMap.tsx

```diff
@@ -63,7 +63,7 @@ export function SwapButton({
 }
 
 const quoteErrorTexts: { [key in GpQuoteErrorCodes]: string } = {
-  [GpQuoteErrorCodes.UNHANDLED_ERROR]: 'Unhandled error',
```

Comment:
nitipic: But does this UNHANDLED_ERROR always mean ERROR_LOADING_PRICES? 

If so, maybe we should rename the enum too.